test/utils/deleteRoles.js

```diff
@@ -0,0 +1,25 @@
+const firestore = require("../../utils/firestore");
+const userCollection = firestore.collection("users");
+
+/**
+ * @return {boolean} success - are roles deleted or not
+ */
```

Comment:
I think we can add comments for description and params for this function

Reply:
Sure. I'll add that